source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/ChoiceSetInputRenderer.java

```diff
@@ -63,7 +63,9 @@ public static ChoiceSetInputRenderer getInstance()
     public View renderCheckBoxSet(
             RenderedAdaptiveCard renderedCard,
             Context context,
-            ChoiceSetInput choiceSetInput)
+            ChoiceSetInput choiceSetInput,
```

Comment:
Nit: Inconsistent formatting of function parameters in teh source code (eg baseacationelementrenderer.java vs this file) . Is it an old school windows style each parameter on a new line? What is the clang formatting rules set to?